apps/list.c

```diff
@@ -958,6 +990,16 @@ static void list_type(FUNC_TYPE ft, int one)
     for (fp = functions; fp->name != NULL; fp++) {
         if (fp->type != ft)
             continue;
+	switch (ft) {
```

Comment:
The indentation for this block looks off.

Reply:
Done. Thanks!